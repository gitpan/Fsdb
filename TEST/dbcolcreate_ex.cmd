prog='dbcolcreate'
args='foo'
cmp='diff -cb '
