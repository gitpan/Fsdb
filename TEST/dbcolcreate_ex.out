#fsdb test foo
a	-
b	-
#   | dbcolcreate foo
